cmd/backend/backend.go

```diff
@@ -20,4 +20,9 @@ var BackendCmd = &cobra.Command{
 
 func init() {
 	cmd.RootCmd.AddCommand(BackendCmd)
+	BackendCmd.PersistentFlags().String("env-name", "dlp", "Name of the conda environment you want to create")
```

Comment:
can we force the env name to be `dlp` for consistency? If so, maybe make the env name a constant?